web/api/report_server.thrift

```diff
@@ -247,6 +247,16 @@ struct BugPathLengthRange {
   2: i64  max, // Maximum value of bug path length.
 }
 
+struct DateInterval {
+  1: i64  before,
+  2: i64  after,
```

Comment:
Is this Unix-time (epoch time, etc.)? https://en.wikipedia.org/wiki/Unix_time
We could document it here.